encode_test.go

```diff
@@ -656,6 +656,17 @@ func (s *S) TestSetIndent(c *C) {
 	c.Assert(buf.String(), Equals, "a:\n        b:\n                c: d\n")
 }
 
+func (s *S) TestSequenceUndent(c *C) {
+	var buf bytes.Buffer
+	enc := yaml.NewEncoder(&buf)
+	enc.SetSequenceUndent(true)
```

Comment:
What is sequence undent?

Reply:
Between yaml v2 and v3 the way sequences is presented changed from 

```
list:
- li1
- li2
```
to
```
list:
  - li1
  - li2
```


Since the default needs to be false in go, I cant simply use `SetSequenceIndent` with a default of true. So `SetSequenceRemoveIndent`, `SetSequenceDedent` and `SetSequenceUnindent` were my other considered names